docs/DeveloperGuide.md

```diff
@@ -376,18 +375,122 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1.  User requests to list orders
-2.  ReadyBakey shows a list of orders
-3.  User requests to clear all saved customers
-4.  ReadyBakey clears all customers
+1. User requests to list orders
+2. ReadyBakey shows a list of orders
+3. User requests to clear all saved customers
+4. ReadyBakey clears all customers
 
-    Use case ends.
+   Use case ends.
 
 **Extensions**
 
 * 2a. The list is empty.
 
-  Use case ends.
+**Use case: Exits the ReadyBakey program**
+
+**MSS**
+
+1. User requests to exit the program.
+2. ReadyBakey says bye and closes the application.
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. User closes the program without using CLI.
+    * 1a1. ReadyBakey closes the application.
+
+**Use case: Locate customer by their name**
+
+**MSS**
+
+1. User requests to find the customer by name in ReadyBakey.
+2. ReadyBakey finds the customers whose names match the requested customer.
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. ReadyBakey detects no customers that match the user's request.
+  * 1a1. ReadyBakey requests for the correct customer name.
+  * 1a2. User enters the correct customer name.
+  * Steps 1a1-1a2 are repeated until the data entered are correct.
+
+    Use case resumes at step 2.
+
+**Use case: Editing customer's address**
+
+**MSS**
+
+1. User requests to edit specific customer's address.
+2. ReadyBakey edits the specified customer's address.
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. ReadyBakey detects no customers that match the user's request.
+    * 1a1. ReadyBakey requests for the correct customer name.
```

Comment:
According to the user guide, we should be identifying users by Index on the display, not customer name. Am not sure if we are going to change it to using customer name instead? 
If we do, there is the issue of repeating names as well, which is why we decided to use phone number for linking an order to a customer right?

I would suggest we stick with the index method that already exists

Reply:
My bad, let me make the changes.